ycsb/main.go

```diff
@@ -206,7 +206,7 @@ func (yw *ycsbWorker) runLoader(n uint64, numWorkers int, thisWorkerNum int, wg
 	if *verbose {
 		fmt.Printf("Worker %d loading %d rows of data\n", yw.workerID, n)
```

Comment:
I've removed this log message and a few others which weren't particularly useful.